src/unix/bsd/apple/mod.rs

```diff
@@ -368,6 +368,13 @@ s! {
         pub fst_bytesalloc: ::off_t,
     }
 
+    pub struct fpunchhole_t {
+        pub fp_flags: ::c_uint, /* unused */
+        pub reserved: ::c_uint, /* (to maintain 8-byte alignment) */
```

Comment:
nit: there is no need to expose this field, its sole role is mentioned in the comment.

Reply:
Oh you're right. If I make these private, can people still instantiate the struct?

Reply:
It might be best to leave them available. I don't believe there's any chance of misuse, and their purposes might change in the future.

Reply:
fair point it s easier to instantiate that way indeed. another note, you can amend your commit message with adding
`close #<issue id>`.

Reply:
I've added that to the one of the commit messages.

Reply:
cool, you just need to rebase into 1 commit and you re good :)